internal/config/config.go

```diff
@@ -461,3 +607,45 @@ func readEnv(key string, def string) string {
 	}
 	return value
 }
+
+// initSSLCertDir update SSL_CERT_DIR to add TLSCAPath if not found
+// in the list of directories.
+// clowderConfig
+func initSSLCertDir(clowderConfig *clowder.AppConfig) {
+	envSSLCertDir := os.Getenv(EnvSSLCertDirectory)
+	if hasTLSCAPath(clowderConfig) {
+		if !checkPathInList(*clowderConfig.TlsCAPath, envSSLCertDir) {
+			envSSLCertDirItems := make([]string, 0, 10)
+			if envSSLCertDir != "" {
+				envSSLCertDirItems = append(envSSLCertDirItems, strings.Split(envSSLCertDir, ":")...)
+			}
```

Comment:
could it be simpler?
```go
if envSSLCertDir != "" {
    envSSLCertDir += ":"
}
envSSLCertDir += *clowderConfig.TlsCAPath
```

This avoids splitting and re-joining the PATH components, and just appends the new element.